controller.java

```diff
@@ -1,14 +1,10 @@
 
 public class controller {
     public static void main(String[] args) {
-        // Declare two numbers
-        int num1 = 10;
-        int num2 = 20;
```

Comment:
change num1 to number1 and vice versa for num2
```suggestion
        int number1 = 10;
        int number2 = 20;
```